pkg/stub/stub.go

```diff
@@ -137,7 +137,8 @@ type PostUpdateContainerInterface interface {
 
 // Stub is the interface the stub provides for the plugin implementation.
 type Stub interface {
-	// Run the plugin. Starts the plugin then waits for an error or the plugin to stop
+	// Run the plugin in blocked way and wait for critical error return from plugin service.
+	// before this function return, stopped plugin can be restart.
```

Comment:
I think this wording is now a bit unclear (especially the 'blocked way' bit). Could we update this to be more clear ? Maybe using something like 
```
// Run starts the plugin then waits for the plugin service to exit, either due to a critical error or an explicit call to Stop(). Once Run() returns, the plugin can be restarted by calling Run() or Start() again.
```

Reply:
already update, thanks